mergify_engine/engine/commands_runner.py

```diff
@@ -196,12 +225,24 @@ async def run_pending_commands_tasks(
             continue
 
         if conclusion == check_api.Conclusion.PENDING:
-            pendings.add(command)
-        elif command in pendings:
-            try:
-                pendings.remove(command)
-            except KeyError:
-                LOG.error("Unable to remove command: %s", command)
+            if comment["created_at"] == comment["updated_at"]:
+                pendings.add(command)
+            else:
+                edited.add(command)
+        else:
+            edited.discard(command)
+            pendings.discard(command)
+
+    for edit in edited:
+        message = prepare_message(
+            f"@Mergifyio {edit}",
```

Comment:
Don't you have the whole text message above ? You could use it instead of reformatting the message here, no ?

Reply:
Good catch, this was even a bug.